argparse/templates/cwltool_inputs.j2

```diff
@@ -1,30 +1,44 @@
 {% extends "cwltool.j2" %}
 
 {% block cwltool_inputs %}
+{% if 'python' in basecommand %}
```

Comment:
That might break when the user does not supply `python` on the commadline, i.e, via shell bang:

```
#!/usr/bin/env python
```

And then invoking the script as-is without the python iterpreter in the argument list:

```
baseCommand: ./script.py
```